section3 dev done - (have made the changes which was asked)

gauthking opened the pull request.

gauthking committed:
section3 dev done

shubhamm69 commented on app/(Home)/Section3.tsx:
```diff
@@ -0,0 +1,51 @@
+'use client'
```
try using a carousel library that supports NextJs Server Side Rendering.
make a server component in directory `components`  for slider and calling it in section tab.
here are a few examples how you can do that so, i had few options in mind. 

here, you can refer [this](https://cloudinary.com/blog/3-ways-to-implement-a-carousel-in-nextjs) and [its sandbox link](https://codesandbox.io/s/brave-gwen-ddfsbg?from-embed=&initialpath=/&file=/pages/index.js).

shubhamm69 commented on package.json:
```diff
@@ -22,9 +22,16 @@
     "react": "18.2.0",
+    "react-alice-carousel": "^2.8.0",
     "react-dom": "18.2.0",
+    "react-responsive-carousel": "^3.2.23",
+    "react-slick": "^0.29.0",
+    "slick-carousel": "^1.8.1",
```
I don't see you using these packages. Please avoid adding unnecessary ones to the final code commit and keep the codebase clean and minimal.

gauthking replied:
removed

shubhamm69 commented on constants/ourTeam.tsx:
```diff
@@ -0,0 +1,91 @@
+import Atharv from '@assets/atharv.png'
+import Aditya from '@assets/aditya.png'
+import Ekta from '@assets/ekta.png'
+import Geo from '@assets/geoffrey.png'
+import Pranshu from '@assets/pranshu.jpeg'
+import Karthik from '@assets/karthik.png'
+import Kushagra from '@assets/kushagra.png'
+import Shrestha from '@assets/shrestha.png'
+import Shubham from '@assets/shubham.png'
+import Sparsh from '@assets/sparsh.png'
+import Tushar from '@assets/tushar.png'
```
Please refrain from adding these images directly to the assets folder; instead, move them to the `assets/team` directory.

gauthking replied:
ok

shubhamm69 commented on constants/ourTeam.tsx:
```diff
@@ -0,0 +1,91 @@
+    image: Atharv,
+    role: 'Secretary',
+    description:
+      'A man with passion and compassion at the same time. Truly a person who can drive multiple forces towards the GOAL. I am sure your abilities will drive Android Club towards a successful and fulfilling journey ahead full of triumph and achievements✨.',
```
We do not use lowercase text anywhere in the project. Please convert all descriptions to uppercase. Using a VS Code shortcut for this can be helpful 😄

gauthking replied:
ok

shubhamm69 commented on app/(Home)/Section3.tsx:
```diff
@@ -0,0 +1,51 @@
+              <Image
+                src={value.image}
+                alt="avatar"
+                className="rounded-[200%] bg-white w-full h-64"
```
also one last thing can u make images size a little smaller, maybe `rounded-full bg-secondary w-full p-5 h-64`

gauthking replied:
done

gauthking committed:
Merge branch 'Android-Club-VITC:main' into main

gauthking committed:
section3 resolved changes asked

gauthking committed:
Merge branch 'main' of https://github.com/gauthking/AC-Website-NextJs

shubhamm69 committed:
minor changes
- removed unused devDependencies
- headings made uppercase
- prettier code formatted
